Dockerfile

```diff
@@ -150,6 +151,50 @@ RUN mkdir -p /vboxguest && \
     mkdir -p $ROOTFS/sbin && \
     cp x86/lib/VBoxGuestAdditions/mount.vboxsf $ROOTFS/sbin/
 
+# Build VMware Tools
+ENV OVT_VERSION 9.4.6-1770165
+
+# Download and prepare ovt source
+RUN mkdir -p /vmtoolsd && cd /vmtoolsd && \
+    curl -L -o open-vm-tools-$OVT_VERSION.tar.gz http://downloads.sourceforge.net/open-vm-tools/open-vm-tools-$OVT_VERSION.tar.gz && \
+    tar xfz open-vm-tools-$OVT_VERSION.tar.gz && rm open-vm-tools-$OVT_VERSION.tar.gz &&\
+    mv open-vm-tools-$OVT_VERSION open-vm-tools
+
+# Apply patches
+RUN cd /vmtoolsd && \
+    curl -L -o open-vm-tools-3.x.x-patches.patch https://gist.github.com/frapposelli/5506651fa6f3d25d5760/raw/a2310970f1b010c57b25d69a5465aa03fe9788fe/open-vm-tools-3.x.x-patches.patch &&\
```

Comment:
Yes, this is definitely temporary, I really wanted to put this gist under our github org but apparently it's not possible yet. In the future we're going to have a specific patches repo in our org.